pftools/python/setup.py

```diff
@@ -23,20 +23,27 @@
     packages=find_packages(),
     install_requires=[
         'pyyaml==5.4',
-        'numpy',
-        'xarray',
-        'numba',
-        'dask',
-        'imageio>=2.9.0'
     ],
     include_package_data=True,
     extras_require={
         'all': [
-            'imageio>=2.9.0',
+            'imageio>=2.9.0'
+            'numpy',
```

Comment:
is numpy core? or can it be truly optional? I don't know, I'm just wondering.

Reply:
I also don't know. I think this is how it was previously.